frontend/common/types/responses.ts

```diff
@@ -508,6 +510,8 @@ export type ChangeRequest = {
 export type FeatureVersion = {
   created_at: string
   updated_at: string
+  feature: number
```

Comment:
This won't necessarily be provided in all scenarios, e.g. creation, update, etc. does it need the `?` like the above?